CMakeLists.txt

```diff
@@ -805,6 +805,7 @@ else()
 endif()
 
 if (WIN32)
+  include_directories(Externals/WIL/include)
```

Comment:
Apparently WIL has a CMakeLists.txt file. Does `add_subdirectory(Externals/WIL)` + `target_link_libraries` against WIL not work?

Reply:
No, I got a whole load of errors with `add_subdirectory(Externals/WIL)`, starting on line 2 (`project(WIL)`). I'm not sure if the CMakeLists.txt file they provide is intended for use by projects including it as a library or only by them. They don't set `include_directories` (except in WIL/tests/CMakeLists.txt), for one.

Reply:
Oh, turns out we are using an older version of WIL that doesn't have the target_include_directories call here :/

https://github.com/microsoft/wil/blob/86353933d5fba0caf61503c8b09e10e182dccdfb/CMakeLists.txt#L36-L40